en/developer_guide/routing.rst

```diff
@@ -0,0 +1,92 @@
+Routing
+=======
+
+Base URL
+--------
+
+In Open Orchestra, the base url of a website is composed by the domain and eventually a language prefix.
+To manage all possibilities, Open Orchestra associate each website in the database with website aliases which contain:
+
+* **domain**: domain name
+* **language prefix**: used by Open Orchestra to define the language
+* **scheme** : to choose with which protocol page can be accessed
+
+Here is an example of some base URL generated for a website associate to two domains: ``dev.example.com`` and ``test.example.com``.
+Each of them, with english as default language and french as secondary one.
+
++----------+--------+------------------+----------+-----------------+-----------------------------+
+| Alias Id | Scheme | Domain           | Language | Language prefix | Base URL                    |
++==========+========+==================+==========+=================+=============================+
+| 0        | http   | dev.example.com  | fr       | fr              | http://dev.example.com/fr   |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+| 1        | http   | dev.example.com  | en       | none            | http://dev.example.com      |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+| 2        | http   | test.example.com | fr       | fr              | http://test.example.com/fr  |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+| 3        | https  | test.example.com | en       | none            | https://test.example.com    |
++----------+--------+------------------+----------+-----------------+-----------------------------+
+
+
+URL Pattern
+-----------
+
+The url pattern is added to the ``base URL`` to generate the url.
+For node, the url pattern can be choose by contributor or it is generated automatically from the page title.
+
+See ``Url pattern`` chapter from `Node Configuration`_ for more details.
+
+Redirection
+-----------
+
+Open Orchestra allows user to determine redirection in the back office.
+The redirection form contains some arguments:
+
+* **site name**: contextual site which trigger redirection
+* **site language**: contextual language which trigger redirection
+* **pattern to redirect**: pattern that will be the URL of the redirection
+* **node where the redirection is headed**: redirect to a node
+* **Url to redirect to**: redirect to url if node is not specify
+* **permanent redirection**: Use to adapt response code
+
+Implementation
+--------------
+
+``open-orchestra-front-bundle`` contains all routing files in the ``routing`` folder.
+
+
+Route Loader
+~~~~~~~~~~~~
+
+In Open Orchestra, ``Route Loader`` provide route from database collections:
```

Comment:
After reading the rest of the paragraph, il think that you should rephrase this sentence